cypress/integration/dropdown.js

```diff
@@ -56,9 +57,11 @@ describe('Dropdown test', function() {
 
         it('Should show all accounts as linked (except GitLab and Bitbucket)', function() {
             everythingOk();
-            cy.visit(String(global.baseUrl) + '/auth/gitlab.com?code=somefakeid', {'failOnStatusCode': false}).then((resp) => {
-                expect(resp.status).to.eq('')
-            })
+            // cy.visit(String(global.baseUrl) + '/auth/gitlab.com?code=somefakeid', {'failOnStatusCode': false}).then((resp) => {
+            //     expect(resp.status).to.eq('')
+            // })
+            // TODO: Gitlab is being very slow, hopefully one day we can remove this
+            cy.wait(10000);
```

Comment:
After commenting it out, it looks like it's checking if everything is ok, then  waiting, then checking if everything is ok again even though nothing has changed